developing-fcc-software/FccSoftwareGit.md

```diff
@@ -14,81 +14,148 @@ For a general introduction to git, have a look at these tutorials:
 -   [Interactive tutorial](https://pcottle.github.io/learnGitBranching/)
 -   [The git book](https://git-scm.com/book/en/v2)
 
-## First time setup of git
+### First time setup of git
+
+Set global information about you, particularly your name and email. This information will be attached to each commit and it will be visible to other contributors. To do it, please use the following commands
+```bash
+git config --global user.name "Name Family-name"
+git config --global user.email "Name.Family-name@cern.ch"
+```
 
 Please refer to [this tutorial](https://git-scm.com/book/en/v2/Getting-Started-First-Time-Git-Setup) and the [GitHub help](https://help.github.com/articles/set-up-git/).
 
-## Generate and set up ssh keys for github
+### Generate and set up ssh keys for github
 
 When working on lxplus we recommend to clone github repositories via SSH, especially if you want to contribute code. For this to work, you need to generate ssh keys for authentication. See the corresponding github [help-page](https://help.github.com/articles/generating-an-ssh-key/).
 
 :::{admonition} Generate and set up ssh keys for github
 :class: callout
 
- If you only want to use the software it may be easier to use https. In that case you don't need to generate the keys but have to replace `git@github:` with `https://github.com/` in all the instructions. Note that you'll not be able to push to your repository when you are on lxplus. You can also start using https for now and later re-add your repository with ssh authentication, see the [trouble shooting section](#trouble-shooting)
```

Comment:
That sentence "Note that you'll not be able to push to your repository" doesn't make much sense now in the middle of this. I think originally what was meant to go here is that if you have your keys locally you can't push from a different place without the keys.